### ISSUE TITLE
Missing global CSS for all components (?)

### ISSUE THREAD

**Issue by thclark** [tailwindlabs/tailwindui-issues#1560]
## TLDR

**Tailwindui documents say nothing about extra global css. Tailwind seems to be installed fine, since some styles are applied. So where does all the extra styling come from?**

## Issue

**What component (if applicable)**
All, although the MRE I've created below uses [the code from this component](https://tailwindui.com/components/application-ui/headings/page-headings#component-d7c74dbf561c0e69da7d009d223bdc0a)

**Describe the bug**
I have followed the installation instructions for [tailwind in nextjs](https://tailwindcss.com/docs/guides/nextjs) and then the additional [requirements for tailwindui](https://tailwindui.com/documentation#requirements) (which is simply the addition of the inter font).

When using code from the examples (I have a paid up subscription, this happens with all that I've used so far), they don't render as expected - they are missing a lot of (what is probably global) css.

**To Reproduce**
I have built a Minimal Reproducible Example [here](https://codesandbox.io/p/devbox/sjct8p) so have at it :)

**Expected behavior**
I expected to be able to drop chunks of code directly into my app and see something like this:
![Screenshot 2024-03-06 at 19 09 35](https://github.com/tailwindlabs/tailwindui-issues/assets/7223170/e5f7b9fd-2457-4f6f-a6c2-35a15d9e07cb)

Instead I see something like this:
![Screenshot 2024-03-06 at 19 07 38](https://github.com/tailwindlabs/tailwindui-issues/assets/7223170/910acd31-4428-4400-a08c-345f9e40da9c)

**Comment by adamwathan** [comment]
Hey! Took a look at the reproduction and it looks like you are disabling all of Tailwind's base styles:

<img width="610" alt="image" src="https://github.com/tailwindlabs/tailwindui-issues/assets/4323180/827083ed-2e66-4ef6-8995-0664aa872338">

If you delete that line, everything should work as expected 👍  Tailwind UI expects a vanilla installation (+ a couple plugins in some cases), so you need to have our Preflight styles in place for things to look as designed.

**Comment by thclark** [comment]
Aha! That's got to be it, ok thank you so much @adamwathan (and kudos on a great framework - great to have issue responses directly from the man himself :) )

For anyone coming across this in the same position, the reason my team had added `preflight: false` when setting up tailwind is because we're migrating an existing enterprise app over to tailwind, one view at a time. I hadn't realised the implication!
